app/views/design-system/rsd-design-system/design-patterns/contact-details/example-contact-per-card-read-only.html

```diff
@@ -0,0 +1,63 @@
+{% extends "./../example-layout.html" %}
+
+{% block pageTitle %}
+Contact details design pattern - contact per card (read only) | RSD component library
+{% endblock %}
+
+{% block sample %}
+<div class="govuk-summary-card">
+  <div class="govuk-summary-card__title-wrapper">
+    <h2 class="govuk-summary-card__title">
+      Project managed by
+    </h2>
+  </div>
+  <div class="govuk-summary-card__content">
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Name
+        </dt>
+        <dd class="govuk-summary-list__value">
+          Alex James
+        </dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Email address
+        </dt>
+        <dd class="govuk-summary-list__value">
+          alex.james@education.gov.uk
```

Comment:
again, don't put emails here, use something generic like example.name@education.gov.uk

Reply:
We have replaced all @education.gov.uk emails with @example.com